legacy/src/api/merchants/merchants.md

```diff
@@ -29,24 +29,38 @@ which define the payment methods available for a Payment Request.
+| createdBy        | {% dt CRN %}       | The User or API Key that created the Merchant.                                                              |
+| updatedAt        | {% dt Timestamp %} | When the Merchant was updated.                                                                              |
+| updatedBy        | {% dt CRN %}       | The User or API Key that updated the Merchant.                                                              |
+| onboardingStatus | String             | The onboarding status of the Merchant. See [Onboarding Statuses](#onboarding-statuses) for possible values. |
 
 {% h4 Optional Fields %}
 
-|      Field       |                  Type                   |                        Description                        |
-| :--------------- | :-------------------------------------- | :-------------------------------------------------------- |
-| test             | Boolean                                 | **EXPERIMENTAL** Flag indicating merchant is for testing. |
-| settlementConfig | [Settlement Config](#settlement-config) | **EXPERIMENTAL** Merchant settlement config               |
-| location         | {% dt Location %}                       | **EXPERIMENTAL** Physical Location of Merchant            |
+|         Field          |                  Type                   |                                Description                                |
+| :--------------------- | :-------------------------------------- | :------------------------------------------------------------------------ |
+| test                   | Boolean                                 | **EXPERIMENTAL** Flag indicating merchant is for testing.                 |
+| settlementConfig       | [Settlement Config](#settlement-config) | **EXPERIMENTAL** Merchant settlement config.                               |
+| location               | {% dt Location %}                       | **EXPERIMENTAL** Physical Location of Merchant.                            |
+| onboardingStatusReason | String                                  | The reason associated with the [Onboarding Status](#onboarding-statuses). |
+
+<a name="onboarding-statuses">
+{% h4 Onboarding Statuses %}
+
+|    Status    |                                                                           Description                                                                            |                                Allowed Reasons                                 |
+| :----------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------- |
+| applied      | Identifies merchants that have registered via the Merchant portal with limited information. Requires Customer Support assistance to progress.                    |                                                                                |
```

Comment:
Will someone looking at our docs know what Customer support is? Seems like something we don't need to mention publicly

Reply:
Ditto for Merchant portal.

We could just say - `Identifies merchants that have registered to be activated for Centrapay payments`